src/App.js

```diff
@@ -95,18 +95,13 @@ class App extends Component {
   render() {
     return (
       <div className="App">
-        {this.state.user ? <Header className="App-header" user={this.state.user} /> : null}
-        <img src={logo} className="App-logo" alt="logo" />
+        <Header user={this.state.user} />
```

Comment:
I am not sure about this change. https://github.com/FromDoppler/doppler-webapp/pull/25/files#r255034909

Let's talk about it.